context/FavContext.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+const Context = React.createContext();
+// Context.Consumer, Context.Provider
```

Comment:
Should we name this something less generic like `TeamContext` if its only going to handle fav teams? or is this context going to handle other global state, and is just handling favteam _for now_?

Reply:
also do we need these comments?